test/__snapshots__/Changelog.spec.js.snap

```diff
@@ -0,0 +1,67 @@
+exports[`createMarkdown get markdown grouped by tags (with commit number link) 1`] = `
+"
+## v0.2.0 (2017-01-01)
+
+#### :rocket: New Feature
+* \`pkg-1\`, \`pkg-2\`
+  * [#1](https://github.com/lerna/lerna-changelog/pull/1) This is the commit title for the issue (#1). ([@test-user](https://github.com/test-user))
+  * [#1](https://github.com/lerna/lerna-changelog/pull/1) This is the commit title for the issue (#1). ([@test-user](https://github.com/test-user))
```

Comment:
Cool! Would be awesome to have a test for 2 tags (New feature/bugfix or something) and see the formatting

basically just the places where the formatting might be different